proj4/meta.yaml

```diff
@@ -7,10 +7,11 @@ source:
   url: http://download.osgeo.org/proj/proj-4.9.1.tar.gz
 
 build:
+  number: 0  # [osx]
   number: 1  # [not osx]
 
-requirements:
-  run:
+requirements:  # [win]
+  run:  # [win]
```

Comment:
> And conda barfs as it thinks we've passed an empty string to the run requirement.

Our geos recipes do have an empty string in the run requirement.  Is that a new "feature" of conda-build?